api/src/db/migration.sql

```diff
@@ -1,12 +1,17 @@
-DROP TABLE IF EXISTS tasks;
+DROP TABLE IF EXISTS attempts;
+DROP TABLE IF EXISTS question_notes;
+DROP TABLE IF EXISTS students;
+DROP TABLE IF EXISTS cohorts;
+DROP TABLE IF EXISTS questions;
+DROP TABLE IF EXISTS users;
```

Comment:
Joe and I couldn't figure out why these didn't exist, I'd be interested if anyone knows why